do.sh

```diff
@@ -249,6 +270,10 @@ function install() {
     init_ovn_fake_multinode
     pull_ovn_fake_multinode
     popd
+    pushd ${topdir}
+    install_ovn_tester
+    pull_ovn_tester
```

Comment:
Does it make sense to move `install_ovn_tester` and `pull_ovn_tester` to `run_test()`?  That way we don't have to run `./do.sh install` every time the tester code changed.

Reply:
Actually I was wrong:
```
STEP 8/8: RUN pip3 install -r /ovn-tester/requirements.txt
--> Using cache 9e62908ad117c9d2ee2f3192ebc38f317064b931a38c4d11ac8203f5f626e602
```
So i'm not sure what's best. :)

Reply:
When I first started this process, I had `install_ovn_tester` and `pull_ovn_tester` as part of `run_test()`. I ended up changing it because my workflow would go like so:
- Make adjustments to `do.sh`
- Run a test
- Wait forever for the tester container to be built
- See a test failure.
- Go to 1

That third step was causing a bunch of stress, and I realized that a lot of the tester container is constant and known at install-time, so I adjusted to building the docker image at install time and running it at runtime.

As you point out, now if there is an error in the tester code, I have to reinstall to re-test it. But if there is a problem in a config file or `do.sh`, re-running the test is fast.

Reply:
Ok, makes sense.  What would you think about adding a section in the readme documenting the steps that need to be done when changing the ovn-tester code?  It doesn't have to be very detailed I guess.